ecommerce/taxes/.mutant.yml

```diff
@@ -10,4 +10,6 @@ matcher:
   ignore:
     - Taxes::Test*
     - Taxes::Configuration*
-    - Taxes::VatRateCatalog#vat_rate_for
\ No newline at end of file
+    - Taxes::VatRateCatalog#vat_rate_for
+    - Taxes::AddAvailableVatRateHandler*
+    - Taxes::RemoveAvailableVatRateHandler*
```

Comment:
I excluded them from mutant because I think the amount of weird tests I would have to write does not justify the benefit we get.